<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>解码小工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_1">
      <attribute name="title">
       <string>自动解码</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="buttonOpenWeb">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(225, 225, 225);</string>
           </property>
           <property name="text">
            <string>开启 WebSocket 服务</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>日志信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="editLog">
           <property name="styleSheet">
            <string notr="true">border:1px solid #333333;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>解码前</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="editDecode_qian">
           <property name="styleSheet">
            <string notr="true">border:1px solid #333333;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>解码后</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="editDecode_hou">
           <property name="styleSheet">
            <string notr="true">border:1px solid #333333;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>手动解码</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
         <item>
          <widget class="QPlainTextEdit" name="editManualDecode_qian">
           <property name="styleSheet">
            <string notr="true">border:1px solid #333333;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="buttonStartDecode">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>1</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(225, 225, 225);</string>
             </property>
             <property name="text">
              <string>开始解码</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonClear">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(225, 225, 225);</string>
             </property>
             <property name="text">
              <string>清除信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="editManualDecode_hou">
           <property name="styleSheet">
            <string notr="true">border:1px solid #333333;
background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
